tests/k8s/1-034_validate_applicationset_reconcile_enabled_set_false/01-errors.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: argocd-test
+  namespace: test
+status:
+  phase: Available
```

Comment:
The ArgoCD should exist and should be in `Available` phase when all the components are disabled.